src/js/structs/Transaction.js

```diff
@@ -1,5 +1,12 @@
+import FormActionTypes from '../constants/TransactionTypes';
+
+const validTypes = Object.values(FormActionTypes);
+
 class Transaction {
-  constructor(path, value, type = 'SET') {
+  constructor(path, value, type = FormActionTypes.SET) {
+    if (!validTypes.includes(type)) {
+      throw new TypeError(`Only the following types are allowed: ${validTypes}`);
```

Comment:
The `validTypes` in the message will be Symbols. Perhaps we should output the `Object.keys(FormActionTypes)`.